internal/tableland/impl/mesa_instrumented.go

```diff
@@ -110,22 +94,6 @@ func (t *InstrumentedTablelandMesa) GetReceipt(
 	return ok, resp, err
 }
 
-// SetController allows users to the controller for a token id.
-func (t *InstrumentedTablelandMesa) SetController(
```

Comment:
nre, after reviewing all these changes I noticed that we don't seem to have been doing relays for locking a controller, even though the contract would have enabled this.